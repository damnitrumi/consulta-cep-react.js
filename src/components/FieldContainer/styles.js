import styled, { css } from "styled-components";

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-direction: column;
    gap: ${theme.spacings.tiny};
    margin-bottom: ${theme.spacings.medium};
  `}
`;
