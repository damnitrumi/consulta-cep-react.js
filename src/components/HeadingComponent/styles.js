import styled, { css } from "styled-components";

export const Container = styled.h1`
  ${({ theme }) => css`
    max-width: auto;
    margin: ${theme.spacings.tiny} 0;
    color: red;
  `}
`;
