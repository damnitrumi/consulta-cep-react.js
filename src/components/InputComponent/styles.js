import styled, { css } from "styled-components";

export const Container = styled.input`
  ${({ theme }) => css`
    padding: ${theme.spacings.tiny};
    outline-color: #0fccff;
    font-size: ${theme.font.sizes.small};
  `}
`;
