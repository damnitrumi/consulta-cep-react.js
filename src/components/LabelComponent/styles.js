import styled, { css } from "styled-components";

export const Container = styled.label`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.medium};
  `}
`;
