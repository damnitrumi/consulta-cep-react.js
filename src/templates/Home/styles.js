import styled, { css } from "styled-components";

export const Container = styled.div`
  ${({ theme }) => css`
    padding: ${theme.spacings.small};
  `}
`;
