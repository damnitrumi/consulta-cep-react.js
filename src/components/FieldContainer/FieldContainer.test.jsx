import { renderTheme } from "../../styles/render-theme";
import { FieldContainer } from ".";

describe("<FieldContainer />", () => {
  it("should render", () => {
    const { container } = renderTheme(<FieldContainer forId="teste" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
