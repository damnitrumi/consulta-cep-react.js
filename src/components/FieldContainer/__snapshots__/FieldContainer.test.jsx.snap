// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FieldContainer /> should render 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 0.8rem;
  margin-bottom: 2.4rem;
}

.c1 {
  font-size: 2.4rem;
}

.c2 {
  padding: 0.8rem;
  outline-color: #0fccff;
  font-size: 1.6rem;
}

<div
  class="c0"
>
  <label
    class="c1"
    for="teste"
  >
    Teste
  </label>
  <input
    class="c2"
    id="teste"
    placeholder="Digite seu CEP no primeiro campo"
    readonly=""
    type="text"
    value=""
  />
</div>
`;
